specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -147,6 +157,49 @@ export function TreeRow({
   const hasNoChildrenNodes =
     nodeStats?.directCount === 0 && nodeStats.childCount === 0;
 
+  const [resource] = useLiveState<SpecifyResource<AnySchema> | undefined>(
+    React.useCallback(() => {
+      const table = genericTables[treeName] as SpecifyTable<AnyTree>;
+      return new table.Resource({ id: row.nodeId });
+    }, [row.nodeId, treeName])
+  );
+
+  const [loadedResource] = useAsyncState(
+    React.useCallback(async () => {
+      if (resource === undefined) return;
+      return hijackBackboneAjax(
+        [Http.NOT_FOUND],
+        async () => resource.fetch(),
+        (status) =>
+          status === Http.NOT_FOUND ? unsafeTriggerNotFound() : undefined
+      );
+    }, [resource]),
+    false
+  );
+
+  const acceptedChildren = loadedResource?.get('acceptedChildren') || '';
```

Comment:
even better, since you only need the name field, we have a `fetchRows` function that would do that even more efficiently as it would let you return just the single field you want rather than the entire resource

Reply:
after some tryouts, went for fetchCollection